lib/ui/page/home/page/chat/widget/chat_forward.dart

```diff
@@ -897,8 +897,14 @@ class _ChatForwardWidgetState extends State<ChatForwardWidget> {
               child: WidgetButton(
                 onPressed: () => MessageInfo.show(
                   context,
-                  id: widget.forwards.first.value.id,
+                  isGroup: widget.chat.value?.isGroup == true,
+                  chatItem: widget.forwards.first.value,
                   reads: reads ?? [],
+                  members:
+                      widget.chat.value?.members
+                          .where((u) => u.user.id != widget.user?.id)
+                          .toList() ??
+                      [],
```

Comment:
Members list here can miss the whole members list. And you will have no ability to fetch more members from such parameters.

Shouldn't `MessageInfo` be more independent? It's a view+controller after all and it can fetch both `Chat` and `ChatItem` by itself, can't it? And you might just pass an ID of message here or something?